src/client/actions/GuildScheduledEventDelete.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const Action = require('./Action');
+const { deletedGuildScheduledEvents } = require('../../structures/GuildScheduledEvent');
+const { Events } = require('../../util/Constants');
+
+class GuildScheduledEventDeleteAction extends Action {
+  handle(data) {
+    const client = this.client;
+    const guild = client.guilds.cache.get(data.guild_id);
+
+    if (guild) {
+      const guildScheduledEvent = guild.scheduledEvents._add(data);
+      if (guildScheduledEvent) {
+        guild.scheduledEvents.cache.delete(guildScheduledEvent.id);
```

Comment:
Just wanted to say how ironic it is... we're adding what we're removing.. Might wanna not cache the result of _add if possible?

Reply:
I really can't recall why I used `_add`  here and not just get it from cache 😾. Replaced it with `getScheduledEvent` in b2ec54e instead. It can emit a partial too now.